docs/node/genesis-doc.md

```diff
@@ -30,9 +30,6 @@ Up to date information about the current genesis file and the current genesis
 document's hash can be found on the Network Parameters page ([Mainnet],
 [Testnet]).
 
-[Mainnet]: mainnet/README.md
-[Testnet]: testnet/README.md
-
 :::
```

Comment:
We usually define the references at the end of the section where they occur for the first time.

```suggestion
:::

[Mainnet]: mainnet/README.md
[Testnet]: testnet/README.md
```